vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailViewModel.kt

```diff
@@ -822,16 +822,20 @@ class RoomDetailViewModel @AssistedInject constructor(
                     if (inReplyTo != null) {
                         // TODO check if same content?
                         room.getTimeLineEvent(inReplyTo)?.let {
-                            room.editReply(state.sendMode.timelineEvent, it, action.text.toString())
+                            viewModelScope.launch {
```

Comment:
I'm afraid this could be cancelled if the ViewModel is destroyed... Same remark for the other change